crates/lsp/src/lsp.rs

```diff
@@ -876,7 +884,7 @@ impl LanguageServer {
             .map(|handlers| {
                 let executor = executor.clone();
                 handlers.insert(
-                    id,
+                    RequestId::Int(id),
```

Comment:
I'm reviewing on mobile, so the context is a bit hard to see - will this require a follow up change for passing string ids?

Reply:
No, I think this should be ok. These are the IDs that are generated by Zed, always integer for now, and I do not see a reason to change it, as both integer and strings are allowed in spec.